src/form-layout-grid.scss

```diff
@@ -69,8 +69,9 @@ $cols: 11;
 
         .fd-form-label {
           display: inline-block;
-          margin-right: 0.5rem;
           padding-bottom: 0.125rem;
+          padding-top: 0.875rem;
+          padding-right: 1rem;
```

Comment:
shouldn't this have RTL, too? I see in RTL it's 0.5rem, not 1rem. Is that correct?

Reply:
updated